action.c

```diff
@@ -1515,7 +1515,7 @@ actionCommit(action_t *__restrict__ const pThis, wti_t *__restrict__ const pWti)
 {
 	actWrkrInfo_t *const wrkrInfo = &(pWti->actWrkrInfo[pThis->iActionNbr]);
 	/* Variables that permit us to override the batch of messages */
-	unsigned nMsgs;
```

Comment:
do you have any specific need for this other than cosmetic? I ask because the variable is always set before being used.

Reply:
code quality checks failed some of the builds when it was without initializer, i think freebsd handled warnings as errors and failed the build without this, so i had to add it.

Reply:
thx, so it looks like some compiler is doing false positives (like in the case 2 lines below).